packages/ui/client/components/ProgressBar.vue

```diff
@@ -0,0 +1,96 @@
+<script setup lang="ts">
+const props = defineProps<{ total: number; failed: number; pass: number; inProgress: boolean }>()
```

Comment:
Why not destructure this props so you can just use the values directly inside template instead of `props.value`

Reply:
total can be outside the computed i.e `const total = unref(total)` since it's been reused in multiple places